src/modules/CreateDrops/TokenDrop/CreateTokenDropSummary.tsx

```diff
@@ -0,0 +1,19 @@
+import { DropSummary } from '../DropSummary';
+
+import { useCreateTokenDropContext } from './CreateTokenDropContext';
+
+export const CreateTokenDropSummary = () => {
+  const { getSummaryData, getPaymentData, createLinksSWR } = useCreateTokenDropContext();
```

Comment:
tried to create a generic context, but not sure how to make this work.

ideally in DropSummary, we can just use the context directly rather than creating individual files like this

DropSummary.tsx
```
const { getSummaryData, getPaymentData, createLinksSWR } = useCreateTokenDropContext(); // useCreateNFTDropContext or useCreateTicketDropContext